test/utils/create_resources.go

```diff
@@ -51,7 +51,16 @@ func RetryWithExponentialBackOff(fn wait.ConditionFunc) error {
 }
 
 func IsRetryableAPIError(err error) bool {
-	return apierrs.IsTimeout(err) || apierrs.IsServerTimeout(err) || apierrs.IsTooManyRequests(err) || utilnet.IsProbableEOF(err)
+	// These errors indicate a transient error that should be retried.
```

Comment:
may indicate -> things like ServerTimeout doesn't necessary mean that it really is transient.

I'm fine with the PR itself (since those are changes only to test functions), but please clarify the comment.

Reply:
Makes sense. Done.